README.md

```diff
@@ -101,7 +101,7 @@ Make sure, that you have defined the Github package Maven repository, in order f
 
 A Nanopore NGS measurement output is delivered to us as a nested folder structure, following this model:
 
-![Nanopore Data Structure Model](./doc/figures/Nanopore_Data_Structure_Model.png)
+![Nanopore Data Structure Model](./doc/figures/Nanopore_Data_Structure_Model.svg)
```

Comment:
NIT: if we have two schemas we need to figures

Reply:
solved